tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/commons/request/ExportAppsRequest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c)  2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.siddhi.distribution.editor.core.commons.request;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Bean class to represent the docker and K8s download request payload.
+ */
+public class ExportAppsRequest {
+    Map<String, String> siddhiApps;
+    String configuration;
+    Map<String, String> variables;
```

Comment:
What's the map named variables contain? Use self-explanatory naming as much as possible